data/assets/scene/solarsystem/missions/juice/fov.asset

```diff
@@ -0,0 +1,9 @@
+asset.require('./fov/gala')
```

Comment:
I think I misunderstood Emma's comment.

So when Janus and the NavCam "look away" from Jupiter, that is not actually a data error.  We currently don't have the intended observing schedule, so I changed those FOV to be always rendered instead.  So when they are looking away from Jupiter/Europa/Ganymede, that is the direction they are actually looking and we don't know at this moment whether it will actually take images at that other location or not